foundations/javascript_basics/clean_code.md

```diff
@@ -268,27 +263,25 @@ function calculateBMI(height, weight) {
 
 This comment helps to refresh the reader on how BMI is calculated in plain English, helping the reader to see why the height needs to be converted and what the following calculation is doing. We are almost there just with the naming but the comment still adds further clarity.
 
-
 ### In conclusion
 
 Now that we've covered these ideas, it's good to return to the reminder we shared at the start. Don't try to write perfectly clean code, this will only lead to frustration. Writing "spaghetti" is inevitable, everyone does it sometimes. Just keep these ideas in mind and with time and patience, your code will start to get cleaner.
 
-Learning to write clean code is a process of constant improvement. One that will extend beyond you _completing_ The Odin Project. This lesson is meant to serve as a primer, a starting point for that journey.
-
+Learning to write clean code is a process of constant improvement. One that will extend beyond you *completing* The Odin Project. This lesson is meant to serve as a primer, a starting point for that journey.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  [This list of clean-code tips](https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/).
-2.  [This article](https://blog.codinghorror.com/coding-without-comments/), [and this one too](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/) about the role of comments in your code.
+1. [This list of clean-code tips](https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/).
```

Comment:
This isn't on you, it's just an issue we have throughout the curriculum, but when it comes to link text, it's super important that it's descriptive enough. The text should either describe where the link will take a user or its purpose. "This list of clean-code tips" isn't terrible, but it can still be improved:

```suggestion
1. Read [10 Principles for Keeping Your Programming Code Clean](https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/) to get some great tips for clean code.
```